ProjectSettings/ProjectSettings.asset

```diff
@@ -12,8 +12,8 @@ PlayerSettings:
   targetDevice: 2
   useOnDemandResources: 0
   accelerometerFrequency: 60
-  companyName: DefaultCompany
-  productName: TapperRedone
+  companyName: Fireball Monkey Studios
```

Comment:
Fixed names as they direct where the files are saved on filesystem.

Reply:
I like the company name, gotta find a logo xD
Product name could be changed, let me know if you have any ideas :)